index.html

```diff
@@ -0,0 +1,68 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" type="text/css">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet">
+    <title>Document</title>
+  </head>
+  <body> 
+    <div class="survey">
+      <div class="top">
+        <h1 id="title">Free  waliyy Coding Form Survey</h1>
+        <p id="description">Thank you for taking the time to help us improve the platform</p>
+      </div>
+      <div class="survey-form">
+        <form id="survey-form">
+          <label for="name" id="name-label">Name</label><br>
+            <input id="name" type="text" placeholder="your name" required><br>
+          <label for="email" id="email-label">Email</label><br>
+            <input id="email" type="email" placeholder="Your email" required><br>
```

Comment:
use the same indentation for `label` and `input` tag

Reply:
- kindly phrase the comments politely, as suggestions instead of orders. For example:
  * This is a polite suggestion: *"I think it would be better to use semantic tags"*
  * This reads as an order: *"Use semantic tags"*
- please include an explanation, a "why", for each requested change. For example:
  * This is a well explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
  * This request lacks a "why": *Why don't you use an article tag here instead of a div?*

Reply:
- kindly phrase the comments politely, as suggestions instead of orders. For example:
  * This is a polite suggestion: *"I think it would be better to use semantic tags"*
  * This reads as an order: *"Use semantic tags"*
- please include an explanation, a "why", for each requested change. For example:
  * This is a well explained request: *Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible*
  * This request lacks a "why": *Why don't you use an article tag here instead of a div?*